quantum/dip_switch.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2018 Jack Humbert <jack.humb@gmail.com>
+ * Copyright 2018 Drashna Jaelre (Christopher Courtney) <drashna@live.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+void dip_switch_update_kb(uint8_t index, bool active);
```

Comment:
I think this feature probably needs keyboard and user init functions, too.

Reply:
Possibly.  I was hoping to avoid that, to be honest, since it's not a great experience. 

I'd rather have the update called for this, which is why I have the "read" in the init.  

However, it does need to be tested, because I'm not sure it works right.

Reply:
Updated this in a way that should forcibly trigger an update for both the individual pins, and the bit-mask.